Dev

JPBP22 opened the pull request.

HenningGC committed:
Actual working API calls

HenningGC committed:
Update test4.ipynb

HenningGC committed:
genres to genre.

HenningGC committed:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

JPBP22 committed:
clean_data_test

made a .ipynb file to clean some of the data, this gets rid of games with empty genres and then adjust the prices.

JPBP22 committed:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

JPBP22 committed:
cleaned_code

JPBP22 committed:
functioning api

pulled 180,000 data points. added them to csv

JPBP22 committed:
changed file name

still needs to pull data correctly

JPBP22 committed:
cleaned api file

changed to create another csv, still needs further debugging

JPBP22 committed:
organized files into folders

JPBP22 committed:
Update steamspy_api_nl.ipynb

JPBP22 committed:
finally pulled all games

JPBP22 committed:
switched folders

JPBP22 committed:
oraganised further

JPBP22 committed:
removed error

JPBP22 committed:
made some changes

steamspy_api is ready for the next call to create all_games_v4 file that should include all the data points we need. I also created a clean_data_v2 file that will be used to later separate the data based on the name column.

JPBP22 committed:
more changes

JPBP22 committed:
Update README.md

JPBP22 committed:
Update README.md

JPBP22 committed:
improved api

still need to add giant bomb api

JPBP22 committed:
Update clean_data_v2.ipynb

JPBP22 committed:
fixes and added new file for giant-bomb api

JPBP22 committed:
Update README.md

JPBP22 committed:
Update README.md

JPBP22 committed:
Update README.md

JPBP22 committed:
deleted unnecessary file

JPBP22 committed:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

JPBP22 committed:
moved files around

JPBP22 committed:
update

HenningGC committed:
GiantBomb API v2

Where headers are added.

JPBP22 committed:
Create setup.py

needs to be  updated with all the libraries

JPBP22 committed:
used cl

HenningGC committed:
added formatter

HenningGC committed:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

JPBP22 committed:
Update setup.py

JPBP22 committed:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

HenningGC committed:
Update gb_api_v2.ipynb

tvintimilla01 committed:
changed file

modified the regular expression for 'demo'

tvintimilla01 committed:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

Commit:
Update clean_data_v2.ipynb

Commit:
Update clean_data_v2.ipynb

Commit:
Update clean_data_v2.ipynb

added description to data filtering

tvintimilla01 committed:
added description titles

descriptions for filters

HenningGC committed:
Update formatter.py

JPBP22 committed:
Update setup.py

miguelgfierro commented on .gitignore:
```diff
@@ -0,0 +1,2 @@
+
+file_cleaned.csv
```
suggestion use `*.csv`

miguelgfierro commented on README.md:
```diff
@@ -1,7 +1,13 @@
+# Important Links 
```
It is better to remove the __pycache__ and do not upload temporal files

miguelgfierro commented on proof_of_previous_work/steam_games.csv:
```diff
@@ -0,0 +1 @@
+Name,Description,Expected Release Day,Characters,DLCs,Concepts,Franchises,Genres,Regions,Releases
```
consider uploading csv files in a separate url

HenningGC committed:
Update formatter.py

JPBP22 committed:
removed data

The data has been removed from this other repository: https://github.com/JPBP22/engine-data

JPBP22 committed:
updated README

JPBP22 committed:
added data files

JPBP22 committed:
Delete Data_Loading_v1.ipynb

JPBP22 committed:
Delete Data_Preprocessing_v1.ipynb

JPBP22 committed:
Delete .DS_Store

AEscF committed:
Add files via upload

AEscF committed:
Delete Data_preprocessing.py

AEscF committed:
Add files via upload

JPBP22 committed:
Add files via upload

HenningGC committed:
Update Data_preprocessing.py

AEscF committed:
Add files via upload

AEscF committed:
Add files via upload

JPBP22 committed:
moved into folders

HenningGC committed:
Create Data Preprocessing.ipynb

miguelgfierro commented on api/formatter.py:
```diff
@@ -0,0 +1,7 @@
+def formatter(game):
```
This PR is massive. A better way of doing this is by splitting into this multiple PRs and have one feature for branch

miguelgfierro commented on api/formatter.py:
```diff
@@ -0,0 +1,7 @@
+def formatter(game):
+    
+    game = game.replace("'","").replace("-"," ").replace(",","").replace(".","").lower()
```
The usual way for two people to work together in the same branch would be via PRs. For example, @JPBP22 started a data prep process, now instead of @HenningGC committing to the same branch, he would do a PR to Jose's branch. If Jose is ok with the code, he would merge the code into his branch, and will ask the rest of the team to review before emerging to main.

Makes sense?

miguelgfierro reviewed (commented):
Some extra comments

HenningGC committed:
Update README.md

JPBP22 committed:
Create README.md

HenningGC committed:
Update README.md

HenningGC committed:
Update README.md

JPBP22 committed:
Add files via upload

JPBP22 committed:
Add files via upload

THIS WAS DONE BY HENNING

JPBP22 committed:
Reomved front and back end

HenningGC committed:
frontend v1

HenningGC committed:
backend v1

HenningGC committed:
Create tests_dataset.py

JPBP22 committed:
Deleted files

JPBP22 committed:
deleted files

HenningGC committed:
Update tests_dataset.py

HenningGC committed:
Update tests_dataset.py

HenningGC committed:
Delete test.py

tvintimilla01 committed:
Update Data_Exploration.py

HenningGC committed:
tests v2

HenningGC committed:
Update package.json

Commit:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

HenningGC committed:
Create tests_model.py

HenningGC committed:
Update README.md

HenningGC committed:
Update README.md

anthonyfyaacoub committed:
Create requirements.txt

anthonyfyaacoub committed:
Delete requirements.txt

AEscF committed:
Data Exploration and Data Loading

New Graphics and Analysis in Data Exploration

JPBP22 committed:
Add or update the Azure App Service build and deployment workflow config

AEscF committed:
Merge branch 'dev' of https://github.com/JPBP22/engine into dev

AEscF committed:
Delete Data_Exploration.py

JPBP22 committed:
Delete frontend-engine-main directory

JPBP22 committed:
Uploaded Front end

HenningGC committed:
backend w chatbot

added chatbot function,
exported recommendations as txt.

HenningGC committed:
Update README.md

HenningGC committed:
Update README.md

AEscF committed:
Create Model_Theoretical_Explanation.ipynb

HenningGC committed:
Update README.md

HenningGC committed:
Update routes.py

JPBP22 committed:
Made some changes added Steammy

Commit:
Create 4-Modelling-v2.ipynb

Updated version of modelling

anthonyfyaacoub committed:
Create requirements.txt

tvintimilla01 committed:
modelling finished

tvintimilla01 committed:
Update 4-Modelling-v2.ipynb

JPBP22 committed:
Update App.css

JPBP22 committed:
further chatbox improvements

HenningGC committed:
callback function fixed

JPBP22 committed:
fixed css issue

AEscF committed:
Update Model_Theoretical_Explanation.ipynb

AEscF committed:
Merge branch 'tests' of https://github.com/JPBP22/engine into tests

HenningGC committed:
new styles

HenningGC committed:
GPT3.5 Functionality successfully added

HenningGC committed:
Delete README.md

JPBP22 committed:
Merge branch 'dev' into backend

JPBP22 committed:
Merge pull request #25 from JPBP22/backend

Merge backend implementation for recommendations and chatbot into dev branch

JPBP22 committed:
Create README.md

JPBP22 committed:
Merge branch 'dev' into frontend

JPBP22 committed:
Merge pull request #26 from JPBP22/frontend

Merge frontend implementation for recommendations and chatbot into dev branch

HenningGC committed:
Moving files from one directory to another

HenningGC committed:
Removed unnecesary files and files that were not supposed to be here

JPBP22 committed:
Create README.md

JPBP22 committed:
Merge pull request #27 from JPBP22/tests

Merge Request: Test

HenningGC committed:
Update README.md

HenningGC committed:
Update README.md

HenningGC committed:
Update README.md

HenningGC committed:
Update README.md

HenningGC committed:
Delete node_modules directory

HenningGC committed:
reorganized files inside directories

HenningGC committed:
Update README.md

JPBP22 committed:
Merge pull request #28 from JPBP22/cloud

Add or update the Azure App Service build and deployment workflow config

anthonyfyaacoub committed:
Create test-models.yml

anthonyfyaacoub committed:
Create test-datasets.yml

anthonyfyaacoub committed:
Update test-models.yml

JPBP22 committed:
Merge pull request #29 from JPBP22/test-workflows

Test workflows

anthonyfyaacoub committed:
Rename .github/test-datasets.yml to .github/workflows/test-datasets.yml

anthonyfyaacoub committed:
Rename .github/test-models.yml to .github/workflows/test-models.yml

anthonyfyaacoub committed:
Update test-datasets.yml

anthonyfyaacoub committed:
Update requirements.txt

JPBP22 committed:
Merge pull request #30 from JPBP22/dev

cloud w dev

HenningGC committed:
Merge pull request #31 from JPBP22/cloud

Merge pull request #30 from JPBP22/dev, theoretically it works (it has been validated), but because we do not have access to the subscription, it won't deploy.
